content/rs/concepts/memory-architecture/memory-management.md

```diff
@@ -4,37 +4,29 @@ description:
 weight: $weight
 alwaysopen: false
 ---
-By default, RES manages a node's memory so data is entirely in RAM for
-performance reasons. The exception to this is RES's [Redis on
-Flash]({{< relref "/rs/concepts/memory-architecture/redis-flash.md" >}})
-feature where Flash memory (SSDs) can be used to store data too. If
-there's not enough RAM available, RES does not allow inserting of more
-data into databases. This limitation is intentional and ultimately
-beneficial. It is detrimental to RES for the OS to take control of
-allocating memory and spill some data to the swap, thus managing memory
-on RES's behalf. RES has been designed to know better than the OS what
-it needs and when it needs it. RES protects the existing data and
-prevents the database from being able to store data into the shards. It
+By default, RES manages node memory so that data is entirely in RAM for improved
+database performance. RES is designed to handle memory management to optimize database 
+performance - better than OS memory management. If not enough RAM is available, 
+RES prevents adding more data into the databases. RES protects the existing data and
+prevents the database from being able to store data into the shards. It 
 can be configured to move the data to another node, or even discard it.
-This depends on [eviction
-policy]({{< relref "/rs/administering/database-operations/eviction-policy.md" >}})
-set on a database by the administrator.
+The behavior depends on the [eviction policy]({{< relref "/rs/administering/database-operations/eviction-policy.md" >}})
+set on each database by the administrator.
 
-With this memory management style, comes the responsibility to monitor
-the nodes, clusters, databases, etc., but you are already doing that,
-right?
+RES [Redis on Flash]({{< relref "/rs/concepts/memory-architecture/redis-flash.md" >}})
+manages memory in a different way so that Flash memory (SSDs) can also be used 
+to store data. 
 
 ## What happens when Redis Enterprise Software is low on RAM?
 
-If free RAM is low, RES will automatically attempt to migrate shards to
-other nodes, (if there are any) to free RAM on this node. If that is not
-possible, RES instructs shards to release memory (which can cause data
-loss if eviction policy allows it, or OOM replies). If shards cannot
-free memory, then RES depends on the OS's OOM killer to kill slaves (but
+If free RAM is low, RES automatically tries to migrate shards to
+other nodes (if there are any) in order to free RAM on this node. If that is not
+possible, RES causes shards to release memory if the eviction policy allows it, 
```

Comment:
rephrase this.
Triggering the eviction can cause data loss IFF the policy allows it, or Redis OOM messages if it doesn't.